lgtm.yml

```diff
@@ -13,6 +13,9 @@ extraction:
       - "libgmock-dev"
       - "dh-exec"
       - "swig3.0"
+      - "uuid-dev"
+      - "libzmq5"
```

Comment:
Can you remove this one? `libzmq3-dev` will bring in `libzmq5` automatically, and in case there's some ABI change and the package name changes, it's one less thing to update here.

Reply:
Updated!